canonical-web-and-design/ubuntu.com
# Build /cloud/private-cloud page

## Done

- Build `/cloud/private-cloud` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cloud/private-cloud
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/19g24YRYzF5dBpUKI1QzTG7fXKj1z0OvwgFll69dFoOs/edit#) and [Design](https://app.zeplin.io/project/6141d0b4743dc928767d815e/screen/6141d0e6a12a9c2971df3d0a).
- [Demo](https://ubuntu-com-10437.demos.haus/cloud/private-cloud)

## Issue / Card

Fixes[ #4410](https://github.com/canonical-web-and-design/web-squad/issues/4410)

## Screenshots
<img width="1440" alt="What is private cloud? | Ubuntu ()" src="https://user-images.githubusercontent.com/57550290/134302272-9091b1f7-d612-4f9a-a2b6-e309c1ef4714.png">
